### PR TITLE
Add tmp folder to gitignore template

### PR DESCRIPTION
**Pull request by LostKobrakai**
https://github.com/elixir-lang/elixir/commit/e935cf078444ccc4291e02c6eb64563884ec4c9a#diff-9e3f7e25c9b25140c73745f6b8b0e7f5aa8c143a5289e33f08a95dcb9b65a6b4R296-R297

### CODE REVIEW

File: installer/templates/phx_single/gitignore [hunk 1]
```diff
@@ -32,3 +32,6 @@ npm-debug.log
 # we ignore priv/static. You may want to comment
 # this depending on your deployment strategy.
 /priv/static/
+
+# Temporary files for e.g. tests
+/tmp
```
Comment (eksperimental) [inline installer/templates/phx_single/gitignore:37]:
```suggestion
/tmp/
```

File: installer/templates/phx_single/gitignore [hunk 2]
```diff
@@ -32,3 +32,6 @@ npm-debug.log
 # we ignore priv/static. You may want to comment
 # this depending on your deployment strategy.
 /priv/static/
+
+# Temporary files for e.g. tests
```
Comment (eksperimental) [inline installer/templates/phx_single/gitignore:36]: In Elixir core, we removed the use of Latin abbreviations to simplify the reading. I think in this case we used "for example"

Comment (LostKobrakai) [reply installer/templates/phx_single/gitignore:36]: This is a 1:1 copy of what is in elixir core though.

Comment (eksperimental) [reply installer/templates/phx_single/gitignore:36]: > This is a 1:1 copy of what is in elixir core though.

Then it should be fixed

Comment (eksperimental) [reply installer/templates/phx_single/gitignore:36]: I think "the agreement" was to do it with all Latin expressions, but we only implemented it for "etc."
The discussion is here: https://github.com/elixir-lang/elixir/pull/9559

Comment (LostKobrakai) [reply installer/templates/phx_single/gitignore]: Seems like it was indeed fixed in elixir core, so I copied the wording from there

**Comment by mcrumm** [comment]
Thanks! <3

**Pull request merged**